backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/optimizations/DataFlowIR.kt

```diff
@@ -71,8 +71,8 @@ internal object DataFlowIR {
 
         abstract class Declared(val isFinal: Boolean, val isAbstract: Boolean, val module: Module?, val symbolTableIndex: Int) : Type() {
             val superTypes = mutableListOf<Type>()
-            val vtable = mutableListOf<FunctionSymbol>()
-            val itable = mutableMapOf<Long, FunctionSymbol>()
+            val vtable = mutableListOf<FunctionSymbol?>()
```

Comment:
Can `null` really get into this vtable and itable? Making these element types nullable is very inconvenient because it has observable impact on other code.